include/lsst/jointcal/PhotometryMapping.h

```diff
@@ -38,11 +38,30 @@ class PhotometryMappingBase {
      * @param[in]  measuredStar  The measured star position to transform.
      * @param[in]  instFlux      The instrument flux to transform.
      *
-     * @return     The on-sky flux transformed from instFlux at measuredStar's
-     *             position.
+     * @return     The on-sky flux transformed from instFlux at measuredStar's position.
      */
     virtual double transform(MeasuredStar const &measuredStar, double instFlux) const = 0;
 
+    /**
+     * Return the on-sky transformed flux uncertainty at (x,y). Identical to
+     * "transform" until freezeErrorScales has been called.
+     *
+     * @param[in]  measuredStar  The measured star position to transform.
+     * @param[in]  instFluxErr   The instrument flux error to transform.
+     *
+     * @return     The on-sky flux transformed from instFlux at measuredStar's position.
+     */
+    virtual double transformError(MeasuredStar const &measuredStar, double instFluxErr) const = 0;
+
+    /**
+     * Once this routine has been called, errors no longer change when parameters are updated.
```

Comment:
This comment is a bit misleading to me.

Looking at the code (and the next comment) I think that after `freezeErrorScales` is called, the current transformation and it's parameters are fixed. Future calls to `transformError` will still transform the errors, but if the `_transfo` parameters are updated those changes will not affect the error transform. Is that right?

If so, I think that needs to be more clear. My interpretation of "errors no longer change when parameters are updated" is that the _errors_ are fixed, not the _transform_.

Reply:
I've cleaned this up and renamed the method `freezeErrorTransform`, which I think is clearer.